### PR TITLE
Update README.md with VDPTag2 Example

### PR DESCRIPTION
**Pull request by ChandlerKilpatrick1**
Adding VDPTag2 as an example!

### CODE REVIEW

**Comment by zsunberg** [comment]
@ChandlerKilpatrick1 did you see INSTRUCTIONS.md?

**Comment by ChandlerKilpatrick1** [comment]
No, I'll follow those instructions and update the PR.

**Review by ChandlerKilpatrick1 — COMMENTED** [review]
Should be good to go! I ran the gen readme function but it produces exactly the same output as the manual changes I made to the readme which is why I did not update that file. @zsunberg

**Comment by rejuvyesh** [comment]
@ChandlerKilpatrick1 tests seem to fail on Travis. Possibly you need to Capitalize the `Project.toml` and `Manisfest.toml`?

**Review by ChandlerKilpatrick1 — COMMENTED** [review]
I changed what you suggested. It should be ready to go now.

**Comment by rejuvyesh** [comment]
Looks like your Manifest file is not correct. It doesn't contain VDPTag2. Which is why travis is failing

File: problems/VDPTag/Project.toml [hunk 1]
```diff
@@ -0,0 +1,27 @@
+name = "VDPTag2"
```
Comment (rejuvyesh) [inline problems/VDPTag/Project.toml]: I think you don't want the same Project.toml from the original project. Rather a new Project.toml with a different uuid.

Comment (MaximeBouton) [reply problems/VDPTag/Project.toml]: The easiest way to generate the Project.toml and the Manifest file is to do as follows: 
- go to `POMDPGallery/problems/YourPOMDP`
- create a `Project.toml` file (empty)
- run `julia --project` 
- add all the packages needed to run `script.jl` using the package manager (all the packages that are imported here for example: https://github.com/JuliaPOMDP/POMDPGallery.jl/blob/master/problems/RockSample/script.jl) 
- Check that `script.jl` runs fine in that environment 
- the Manifest.toml file should be created automatically.

**Comment by ChandlerKilpatrick1** [comment]
Code is running locally with my updated Project and Manifest files. I'll keep an eye on this to make sure the builds pass as well.

**Comment by ChandlerKilpatrick1** [comment]
Hey @rejuvyesh, this looks like it's ready to merge.

**Comment by rejuvyesh** [comment]
Thanks a bunch!

**Pull request merged**